nemo/collections/tts/modules/common.py

```diff
@@ -232,24 +231,36 @@ def masked_conv_to_sequence(self, context: Tensor, lens: Tensor, enforce_sorted:
         )
         return seq
 
-    def forward(self, context: Tensor, lens: Optional[Tensor] = None) -> Tensor:
-        if lens is None:
-            for conv in self.convolutions:
-                context = self.dropout(F.relu(conv(context)))
-            context = context.transpose(1, 2)
-            context, _ = self.bilstm(context)
+    def forward(self, context: Tensor, in_lens: Optional[Tensor]) -> Tensor:
```

Comment:
typechecking probably expects "lens" not "in_lens", plus it should be Optional[Tensor]

Reply:
What type checking? This is not a Nemo Module. And it is Optional.